### ISSUE TITLE
Harvestella (1816300)

### ISSUE THREAD

**Issue by amatrelan** [ValveSoftware/Proton#6614]
# Compatibility Report
- Harvestella
- 1816300

## System Information
Two systems:

Steam Deck
- OS: SteamOS Holo
- OS Version: 3.4.4
- OS Build: 20221228.1
- Kernel: 5.13-valv-36-1-neptune
- Steam version: 1676336721
- Driver: Mesa 22.2.0

Desktop:
- GPU: NVIDIA TU104 [GeForce RTX 2070 SUPER] 
- Driver: nvidia v: 525.89.02
- Kernel: 6.2.2-zen2-1-zen
- Proton Version: Default and Experimental
- Full system info [gist](https://gist.github.com/Amatrelan/0630002d551e07cf4747baf76b2013c4)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1816300.log](https://github.com/ValveSoftware/Proton/files/10956920/steam-1816300.log) this is on my desktop and it gets stuck.

I let it enter to cutscene and after while I quit game, not from menu but "force quit".

I think most relevant line in there is:
`winegstreamer: failed to create h264parse, are 64-bit GStreamer "base" plugins installed?`
h264parse is not installed even thought gst-plugins-base is installed.

## Symptoms 
When prerendered cutscene is supposed to start it skips it. 
On proton experimental it gets stuck on that and in default version what game has it skips and game continues.

## Reproduction
1. Start new game
2. Play about 5min into game and there starts pre-rendered cutscene.
3. When scene starts it skips it on default and with experimental it gets stuck.
4. No cutscene on either version.

**Comment by amatrelan** [comment]
This is steam deck log
- Proton version: proton-7.0-6e

Steam deck [proton log](https://gist.github.com/Amatrelan/44790b9f4e18bf1b3edec9961114e739)
With this one it skips that video completely

**Comment by alasky17** [comment]
@Amatrelan Could you give more details about what is happening in the game immediately before the troubling cutscene?  I want to be sure I'm looking at the right thing :)

**Comment by amatrelan** [comment]
About 5 minutes from beginning of game you there is earthquake and after that you go outside house and there is few dialogues and screen goes black (in this point there should play prerendered video).

I'm my PC it just gets stuck on that black screen, but FX sounds continue to play.

On steam deck it skips that video completely and continues

**Comment by IntraDemonic** [comment]
I also have this issue, it occurs on every version of Proton I have tried. I suspected it was missing gstreamer plugins, but after installing both 64 and 32 bit versions of all of the available plugins (on Arch and AUR) it still happens. 
[steam-1816300.log](https://github.com/ValveSoftware/Proton/files/11062815/steam-1816300.log)
Here's my log file, if more examples is helpful

I'm sure it happens with any pre-rendered cutscene, but I specifically have it with the cutscene that plays as part of the initial tutorial/intro when the earthquake happens and you go outside. The characters have a few more text boxes, then I assume it goes to a pre-rendered cutscene, but for me the screen goes black and the audio continues as it was before the cutscene started. I tried to wait it out, in case it would return to the game after the runtime of the video but it never recovered.

I don't have a Steam Deck, but my setup is:
GPU: Nvidia GTX 1080
Driver: 525.89.02-11
Kernel: 6.2.6.arch1-1
Proton version: Occurs on Experimental, 7.0-6, 6.3-8, GE-Proton7-35, GE-Proton7-51

Common to the OP, I use an AMD CPU. Specifically, I have a Ryzen 5 5700G.

**Comment by alasky17** [comment]
@Amatrelan @IntraDemonic Thank you for the details :) This is certainly a video playback failure as you both described.  Unfortunately, it is potentially due to a bug in wine, and thus I'm not completely surprised that trying to install gstreamer plugins would not work (although that would have been nice).  Unfortunately, it looks like this is not a regression or something easily fixed by enabling h264, and so there isn't anything I can do right now.  The difference in behavior is actually likely due to some improvement that is making the behavior closer to Windows, although it is not all the way there.  I don't have a timeline on when this specific game will be fixed, but we are constantly working to improve video playback support in upstream Wine and in Proton, so I have high hopes that at some point the videos in this game will play correctly.

**Comment by Michael5011st** [comment]
To add to this. this issue only started sometime between the beginning of march and now

**Comment by alasky17** [comment]
@Michael5011st Unfortunately, in that case it seems likely that a game update changed the way video playback works within the game :(

**Comment by Michael5011st** [comment]
Good point there was a patch february 7th and it did modify one of the prerendered cutscenes to not be blacked out on the nintendo switch https://www.reddit.com/r/HARVESTELLA/comments/10wfea9/the_harvestella_patch_today_makes_it_super_easy/

**Comment by Michael5011st** [comment]
I wonder if that fix broke something on the pc side for proton

**Comment by Michael5011st** [comment]
@alasky17 the newest proton experimental build has fixed the prerendered cutscenes

**Comment by amatrelan** [comment]
Yes, seems to be so in Steam Deck that `proton experimental` fixed cutscenes, I don't know if this also fixes in Linux PC (cannot access right now to my PC).
